### DISCUSSION TITLE
coqui linux set up

### DISCUSSION THREAD

**Discussion by Nanayeb34** [coqui-ai/TTS#2376] [General Q&A]
hello guys
Any help on how to setup coqui locally for ubuntu.  I want to use the model from the command line.
 I have tried running the code provided in the read me but after installing the repo, it returns the following error when I attempt to view the installed models using tts --list models
Command 'tts' not found, did you mean:
  command 'tst' from deb pvm-examples (3.4.6-3.2)
  command 'pts' from deb openafs-client (1.8.8.1-3ubuntu2~22.04.1)
  command 'tty' from deb coreutils (8.32-4.1ubuntu1)
  command 'ttv' from deb ttv (3.107-1.1)
  command 'tcs' from deb tcs (1-11.1)
  command 'bts' from deb devscripts (2.22.1ubuntu1)
  command 'trs' from deb konwert (1.8-13.2)
  command 'tth' from deb tth (4.15+ds-1)
  command 'ttm' from deb ttm (4.15+ds-1)
  command 'ttx' from deb fonttools (4.29.1-2build1)
  command 'ots' from deb ots (0.5.0-6)
  command 'hts' from deb httptunnel (3.3+dfsg-4)
  command 'tt' from deb treetop (1.6.8-1)
  command 'ts' from deb moreutils (0.66-1)
  command 'tqs' from deb ssake (4.0.1-1)
Try: sudo apt install <deb name>

**Comment by francois-vz** [comment]
You didn't install it right. I don't like the pip install, so rather use the following instructions:

```bash
$ bash # clean terminal, else create conda env
$ git clone https://github.com/coqui-ai/TTS
$ pip install -e .[all,dev,notebooks]  # Select the relevant extras
$ make system-deps
$ make install
```

After that just call as follows

```bash
 $ tts --text "<insert-your-string-here>" --model_path "tts_models/en/ljspeech/speedy-speech" --out_path test.wav 
```

**Reply by Nanayeb34** [reply]
Thanks for your response, but I didn't use pip install TTS . I cloned the repo and run the commands you've outlined.
